app/Resources/views/session/show.html.twig

```diff
@@ -3,6 +3,7 @@
 {% set currentUrl = url('session_show_by_slug', {'id': session.id, 'slug': session.slug}) %}
 
 {% block title %}{{ session.title }} | {{ parent() }}{% endblock %}
+{% block og_title %}{{ session.title }}{% endblock %}
```

Comment:
because this block will be used as an attribute of an element we should escape this. 
`{{ session.title|e('html_attr') }}`